web/app.js

```diff
@@ -1850,7 +1858,11 @@ function handleMouseWheel(evt) {
 
     var previousScale = pdfViewer.currentScale;
 
-    PDFViewerApplication[direction](Math.abs(ticks));
+    if (delta < 0) {
+      PDFViewerApplication.zoomIn(-delta / MOUSE_WHEEL_ZOOM_FACTOR);
```

Comment:
I think that `zoomIn/zoomOut` got switched around here (and below), since the zoom direction is now incorrect. E.g. scrolling up now zoom out, instead of in (and vice versa).